src/components/DefineCompany/FolioNumber.vue

```diff
@@ -37,13 +37,19 @@ import { Component, Emit, Prop, Vue, Watch } from 'vue-property-decorator'
 export default class FolioNumber extends Vue {
   // Props
   @Prop()
-  private initialValue!: string
+  private initialValue: string
 
   @Prop({ default: false })
-  private isEditing!: boolean
+  private isEditing: boolean
 
   // Data variables
-  private folioNumber:string = null
+  private folioNumber: string = null
+
+  // Validation rules
+  private readonly rules: Array<Function> = [
+    v => /^[0-9A-Za-z]*$/.test(v) || 'Invalid character', // numbers and letters only
+    v => (!v || v.length <= 50) || 'Cannot exceed 50 characters' // maximum character count
```

Comment:
Cool, i'm gonna use this, thanks!

Reply:
Natch. Just note that it's `!v || v.length <= 50` if v is optional, or `v?.length <= 50` if v is required.